CARDS-2728: [YE] Add a statement at the top of the YVM and PMOO surveys and in the emails to clarify that everyone visiting the Centre receives it regardless of their diagnosis

marta- opened the pull request:
- Removed the word "cancer" before "treatment" from the statement added previously to the YVM welcome screen
- Added statement to:
  - PMOO and PMOO-IP welcome screens
  - YVM, PMOO, and PMOO-IP emails
- Updated the estimated completion time for PMOO and PMOO-IP on the welcome screens and in the emails

marta- committed:
CARDS-2728: [YE] Add a statement at the top of the YVM and PMOO surveys and in the emails to clarify that everyone visiting the Centre receives it regardless of their diagnosis

- Removed the word "cancer" before "treatment" from the statement added previously to the YVM welcome screen
- Added statement to:
  - PMOO and PMOO-IP welcome screens
  - YVM, PMOO, and PMOO-IP emails
- Updated the estimated completion time for PMOO and PMOO-IP on the welcome screens and in the emails

marta- committed:
CARDS-2728: Update YVM, PMOO, and PMOO-IP welcome screens

marta- committed:
CARDS-2728: Update PM-OO-IP email templates

marta- committed:
CARDS-2728: Update PM-OO email templates

marta- committed:
CARDS-2728: Update YVM email templates

acrowthe commented on prems-resources/clinical-data/src/main/resources/SLING-INF/content/apps/cards/clinics/PM-OO/mailTemplates/InitialInvitation/bodyTemplate.html:
```diff
@@ -41,10 +41,10 @@
+      We have done our best to ensure this survey does not reach families
+      who have lost a loved one. If you received it after a loss,
+      please accept our sincere condolences and apologies.
+      You may complete the survey on behalf of your loved one if you wish.
```
This line is different from what is in other mails - is that intentional?
The others are
> You are welcome to complete the survey on their behalf if you wish.

marta- replied:
Right, it should be like the others
```suggestion
      You are welcome to complete the survey on their behalf if you wish.
```

marta- replied:
Amended the commit labeled `CARDS-2728: Update PM-OO email templates` for a cleaner history.

acrowthe commented on prems-resources/clinical-data/src/main/resources/SLING-INF/content/apps/cards/clinics/PMH-YVM/mailTemplates/InitialInvitation/bodyTemplate.html:
```diff
@@ -4,8 +4,16 @@
+    <p>
+      This survey is sent to all patients who have recently had an appointment at Princess Margaret Cancer Centre.
+      Your appointment may have been for an assessment, for treatment, or for follow-up after your treatment.
+      This survey is to evaluate your experience being at the Cancer Centre.
```
```suggestion
      This survey is to evaluate your experience at the Cancer Centre.
```

marta- replied:
For these other changes we should check with PMPE @Hajany .

acrowthe commented on prems-resources/clinical-data/src/main/resources/SLING-INF/content/apps/cards/clinics/PMH-YVM/mailTemplates/InitialInvitation/bodyTemplate.txt:
```diff
@@ -1,9 +1,14 @@
+
+This survey is sent to all patients who have recently had an appointment at Princess Margaret Cancer Centre.
+Your appointment may have been for an assessment, for treatment, or for follow-up after your treatment.
+This survey is to evaluate your experience being at the Cancer Centre.
```
```suggestion
This survey is to evaluate your experience at the Cancer Centre.
```

acrowthe commented on prems-resources/clinical-data/src/main/resources/SLING-INF/content/apps/cards/clinics/PMH-YVM/mailTemplates/ReminderNotification/bodyTemplate.html:
```diff
@@ -4,10 +4,18 @@
+    <p>
+      This survey is sent to all patients who have had an appointment at Princess Margaret Cancer Centre.
+      Your appointment may have been for an assessment, for treatment, or for follow-up after your treatment.
+      This survey is to evaluate your experience being at the Cancer Centre.
```
```suggestion
      This survey is to evaluate your experience at the Cancer Centre.
```

acrowthe commented on prems-resources/clinical-data/src/main/resources/SLING-INF/content/apps/cards/clinics/PMH-YVM/mailTemplates/ReminderNotification/bodyTemplate.txt:
```diff
@@ -1,10 +1,16 @@
+
+This survey is sent to all patients who have recently had an appointment at Princess Margaret Cancer Centre.
+Your appointment may have been for an assessment, for treatment, or for follow-up after your treatment.
+This survey is to evaluate your experience being at the Cancer Centre.
```
```suggestion
This survey is to evaluate your experience at the Cancer Centre.
```

acrowthe commented on prems-resources/clinical-data/src/main/resources/SLING-INF/content/Survey/YVM.xml:
```diff
@@ -28,20 +28,18 @@
 **This survey is sent to all patients who have had an appointment at Princess Margaret Cancer Centre.**
-Your appointment may have been for an assessment, for cancer treatment, or for follow-up after your treatment.
+Your appointment may have been for an assessment, for treatment, or for follow-up after your treatment.
 This survey is to evaluate your experience being at the Cancer Centre.
```
```suggestion
This survey is to evaluate your experience at the Cancer Centre.
```

Hajany reviewed (approved):
PM approved these changes (final versions in JIRA issue OneDrive). If we're removing the word "being" from this line "This survey is to evaluate your experience being at the Cancer Centre" then also do this on the Welcome Screen

marta- committed:
CARDS-2728: Remove "being" from the statement added to the YVM emails  and welcome screen

"This survey is to evaluate your experience being at the Cancer Centre." -> "This survey is to evaluate your experience at the Cancer Centre."

Co-authored-by: acrowthe <acrowthe3388@gmail.com>
